daml/splice-dso-governance-test/daml/Splice/Scripts/TestValidatorLicense.daml

```diff
@@ -195,3 +195,122 @@ test_ValidatorLivenessWeightInRunNextIssuanceD = do
       abort ("Bob should receive ~15000 times rewards. Expected within: [" <> show bobExpectedMin <> ", " <> show bobExpectedMax <> "], Got: " <> show bobReward)
 
     return ()
+
+-- Test multiple validator license weight changes can be done via single vote
+testValidatorLicenseWeightChangeBatch : Script ()
+testValidatorLicenseWeightChangeBatch = do
+  (app, dso, (sv1, sv2, sv3, sv4)) <- initDevNet
+
+  validator1 <- setupValidator app "validator1"
+  validator2 <- setupValidator app "validator2"
+  validator3 <- setupValidator app "validator3"
+
+  [(dsoRulesCid, _)] <- query @DsoRules dso
+
+  -- Verify all validators have default weight (None)
+  [(_, license1)] <- query @ValidatorLicense validator1
+  [(_, license2)] <- query @ValidatorLicense validator2
+  [(_, license3)] <- query @ValidatorLicense validator3
+  license1.weight === None
+  license2.weight === None
+  license3.weight === None
+
+  -- Change weights for all three validators in a single vote
+  initiateAndAcceptVote app [sv1, sv2, sv3, sv4] (ARC_DsoRules (SRARC_ValidatorLicense_ChangeWeight
+    (DsoRules_ValidatorLicense_ChangeWeight with
+      updates = [ValidatorWeightUpdate validator1 5.0
+               , ValidatorWeightUpdate validator2 10.0
+               , ValidatorWeightUpdate validator3 0.0])))
+
+  -- Verify three instruction contracts were created
+  instructions <- query @ValidatorLicense_Instruction dso
+  length instructions === 3
+
+  -- Execute all three instructions
+  [(license1Cid, _)] <- query @ValidatorLicense validator1
+  [(license2Cid, _)] <- query @ValidatorLicense validator2
+  [(license3Cid, _)] <- query @ValidatorLicense validator3
+
+  [(instruction1Cid, _)] <- queryFilter @ValidatorLicense_Instruction dso (\i -> i.validator == validator1)
+  [(instruction2Cid, _)] <- queryFilter @ValidatorLicense_Instruction dso (\i -> i.validator == validator2)
+  [(instruction3Cid, _)] <- queryFilter @ValidatorLicense_Instruction dso (\i -> i.validator == validator3)
+
+  submitMulti [sv1] [dso] $ exerciseCmd dsoRulesCid DsoRules_Execute_ValidatorLicense_Instruction with
+    sv = sv1
+    validatorLicenseCid = license1Cid
+    instructionCid = instruction1Cid
```

Comment:
consider packaging the instruction execution into a function in `DsoTestUtils`. Something like `executeAllVoteExecutionInstructions`

Reply:
Done